pkg/logging/console.go

```diff
@@ -170,6 +170,9 @@ func (enc *ConsoleEncoder) EncodeEntry(ent zapcore.Entry, fieldList []zapcore.Fi
 		} else {
 			padding -= lineLength
 		}
+		if padding < 0 {
+			padding = 0
+		}
```

Comment:
I suppose this can only happen if `size.Width < printableWidth(fields.String())` - which _should_ only happen if the fields are really long. If this is more a property of the terminal, we might need to look into `TermSize` and why it's not returning a reasonable fallback size.

Reply:
I looked into this at one point and it seems like the unix syscall it uses to determine window size returns 0s when debugging in Intellij (it works fine when running with debugging disables).

```go
_, _, errno := unixSyscall(
unix.SYS_IOCTL,
fp.Fd(),
uintptr(unix.TIOCGWINSZ),
uintptr(unsafe.Pointer(&ws)))
```

Reply:
Perhaps another thing we could do then is check for 0s and default to the 80x60 in that case as well.

Reply:
Eg, by updating the switch case to
```go
	case err != nil, termSize.Width == 0 && termSize.Height == 0:
		fmt.Fprintf(os.Stderr, "Could not get terminal size: %v\n", err)
		termSize = tsize.Size{
			Width:  80,
			Height: 60,
		}
```

Reply:
Right. I think some version of this fix had that exact change as well. It just got lost as it's been punted from branch to branch on my local machine.

Reply:
Just added this change to this PR.